README.md

```diff
@@ -100,13 +100,16 @@ See [Addon / Framework Support Table](ADDONS_SUPPORT.md)
 
 ## Live Examples
 
-### 3.4.alpha
+### 4.0.alpha
 > Note, this is an Alpha version. Some of the features still might not be released
 
-- [React Official](https://storybooks-official.netlify.com)
-- [Vue](https://storybooks-vue.netlify.com/)
-- [Angular](https://storybooks-angular.netlify.com/)
-- [Polymer](https://storybooks-polymer.netlify.com/)
+TBD
+
+### 3.4
+- [React Official](https://release-3-4--storybooks-official.netlify.com)
+- [Vue](https://release-3-4--storybooks-vue.netlify.com/)
+- [Angular](https://release-3-4--storybooks-angular.netlify.com/)
+- [Polymer](https://release-3-4--storybooks-polymer.netlify.com/)
 
 ### 3.3
```

Comment:
Let's remove it. I think it only make sense to have links to latest and alpha versions here

Reply:
I feel like for people with the older versions it will be a useful thing. At least to compare what they are missing, or how something changed...

Reply:
BTW, are we remaining on the "master as a release branch" strategy? I just can add the links to muster instead of the TBD.

Reply:
No, we've decided that master will be 4.0

Reply:
Now you can

Reply:
👌